app/src/main/java/org/mozilla/focus/browser/cursor/CursorController.kt

```diff
@@ -35,12 +36,11 @@ class CursorController(
         cursorParent: View,
         private val view: CursorView
 ) : AccessibilityManager.TouchExplorationStateChangeListener, LifecycleObserver {
-    var isEnabled = true
-        set(value) {
-            field = value
-            keyDispatcher.isEnabled = value
-            view.visibility = if (value) View.VISIBLE else View.GONE
-        }
+
+    private var isEnabled: Boolean by Delegates.observable(true) { _, _, newValue ->
```

Comment:
My understanding is that `Delegates.observable` is intentionally designed so that you can't modify the value.

`property` is not the backing field, `field` (which is what I think you mean by "internal variable"): it's an object that represents Kotlin's conception of a property and lets you inspect it (e.g. it lets you get the backing field with `javaField`, or the wrapping getter with, `javaGetter`, or state like `isInit`). https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.reflect/-k-property/index.html To be honest, I'm not sure what the use case is for passing it in.

If you wanted to change the value, you can:
- Create a raw setter function (as we did before)
- Use Delegates.vetoable, which allows you to "veto" a newValue (rather than change it to whatever you want, as you can with the raw setter)